src/components/routes/Host/ListingForm/ListingForm.tsx

```diff
@@ -220,6 +223,23 @@ class ListingForm extends React.Component<Props, State> {
   }
 
   setNextCrumb = (nextCrumb: string) => this.setState({ nextCrumb });
+
+  handleSubmit = (values: ListingInput, actions: FormikActions<FormValues>) => {
+    const { props } = this;
+    actions.setSubmitting(true);
+    const { updateListing } = props;
+    const { id } = props.match.params;
+    return updateListing(id, values)
+      .then(() => {
+        props.history.push(`/host/listings/${this.state.nextCrumb}`);
+      })
+      .catch((error: ApolloError) => {
+        const formattedError = error.graphQLErrors ? error.graphQLErrors.map(e => e.message).join('\n').toString() : error;
```

Comment:
i think `/n` is normally for unix/max osx +
https://stackoverflow.com/questions/15433188/r-n-r-and-n-what-is-the-difference-between-them/15433225#15433225

you might want to use a combination of `/n` and `/r` to hit windows